bootstrap.sh

```diff
@@ -64,3 +64,23 @@ do
     && cp -n Local.templates/$CONFIG_FILE Local/$CONFIG_FILE \
   )
 done
+
+# Build Yubikit
+YUBIKIT_DIR=Carthage/Checkouts/yubikit-ios
+YUBIKIT_OUT=ThirdParty/YubiKit
+
+rm -rf YUBIKIT_DIR
+rm -rf YUBIKIT_OUT
+
+if [ ! -d "$YUBIKIT_DIR" ]; then
+  git clone https://github.com/jumde/yubikit-ios/ $YUBIKIT_DIR
+fi
+
+SRCDIR=$PWD
+mkdir -p $SRCDIR/$YUBIKIT_OUT/include
+cd $YUBIKIT_DIR/
+git checkout simulator_release
+cd YubiKit
+sh build.sh
```

Comment:
This prints out too much output and Travis hits the maximum log level before the build completes. Either:
- Write to a log file and if an error occurs, print log file destination. (`sh build.sh > file.log 2>&1`)
- Dump all stdout to /dev/null (`sh build.sh > /dev/null`), allowing only errors to surface in `stderr`

Reply:
resolved!